integration-tests/pom.xml

```diff
@@ -147,10 +78,47 @@
     <module>cucumber-tests-protocol-iec60870</module>
 
     <module>cucumber-tests-platform-common</module>
-    <module>cucumber-tests-platform-distributionautomation</module>
-    <module>cucumber-tests-platform-publiclighting</module>
-    <module>cucumber-tests-platform-smartmetering</module>
-    <module>cucumber-tests-platform-microgrids</module>
-
   </modules>
+
+  <profiles>
+    <profile>
+      <id>flexovl</id>
```

Comment:
The mechanism of separating builds into profiles looks good.
I am a little doubtful regarding the naming of the flexovl and smhe profiles.
The naming feels based on particular applications outside the scope of the platform.
In the context of GXF would it be better to choose names reflecting the domains instead (public lighting, smart metering, possibly abbreviated to something like pl and sm to ease typing the profile flags on the command line)?

Reply:
As discussed: changed to 'pl'